core/common/src/main/java/alluxio/resource/ResourcePool.java

```diff
@@ -129,14 +134,15 @@ public T acquire(Integer time, TimeUnit unit) {
         mTakeLock.unlock();
       }
     } catch (InterruptedException e) {
-      throw new RuntimeException(e);
+      throw new IOException(e);
```

Comment:
Following it is a section of getBlockLock(long blockId) in BlockLockManager.class. An unchecked exception will break the designs.
        // Since a block lock hasn't already been allocated, try to acquire a new one from the pool.
      // Acquire the lock outside the synchronized section because #acquire might need to block.
      // We shouldn't wait indefinitely in acquire because the another lock for this block could be
      // allocated to another thread, in which case we could just use that lock.
        blockLock = mLockPool.acquire(1, TimeUnit.SECONDS);
        if (blockLock != null) {
          synchronized (mSharedMapsLock) {
            // Check if someone else acquired a block lock for blockId while we were acquiring one.
            if (mLocks.containsKey(blockId)) {
              mLockPool.release(blockLock);
              blockLock = mLocks.get(blockId);
            } else {
              mLocks.put(blockId, blockLock);
            }
            blockLock.addReference();
            return blockLock;
          }
        }